src/SprykerEco/Zed/Inxmail/Persistence/InxmailQueryContainer.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+/**
+ * MIT License
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+namespace SprykerEco\Zed\Inxmail\Persistence;
+
+use Spryker\Zed\Kernel\Persistence\AbstractQueryContainer;
+
+/**
+ * @method \SprykerEco\Zed\Inxmail\Persistence\InxmailPersistenceFactory getFactory()
+ */
+class InxmailQueryContainer extends AbstractQueryContainer implements InxmailQueryContainerInterface
```

Comment:
As I know we don't use `QueryContainer` anymore, at least in new modules. We are using `EntityManager` and `Repository` instead.

Reply:
Removed